File: geometry_basics_distance_between_points_in_2d/gbdbpi2.js
function distanceBetweenPoints(a, b) 
{
const
{
x: x1, y: y1
} = a;
const
{
x: x2, y: y2
} = b;
return Math.sqrt((x2 - x1) ** 2 + (y2 - y1) ** 2);
}

File: geometry_basics_distance_between_points_in_2d/gbdbpi2.cs
// [geometry basics distance between points in 2d]
// [8 kyu]
// [c#]
using System;
public class Kata
{
public static double DistanceBetweenPoints(Point a, Point b)
{
double squaredDistanceY = Math.Abs(a.Y - b.Y);
double squaredDistanceX = Math.Abs(a.X - b.X);
double distance = Math.Sqrt(Math.Pow(squaredDistanceY, 2) + Math.Pow(squaredDistanceX, 2));
return distance;
}
}
